docs/contributing.md

```diff
@@ -10,35 +10,36 @@ The following is a set of rules for contributing to **Rucio** and its
 packages. Use your best judgment, and feel free to propose changes to this
 document.
 
-If you have questions, you can reach the core development team on our 
-[__Mattermost__](mattermost.md) channel, or send an email to our
-development mailing list [__rucio-dev@cern.ch__](mailto:rucio-dev@cern.ch).
+If you have questions, you can reach the development team on our 
+[__Mattermost__](mattermost.md) channel.
 
 ## What should I know before I get started
 
-A contribution can be either be a **patch** or **feature**:
+A contribution can be either be included in the **next** major release or the
+ongoing **current** release line, e.g. `release-33` This becomes relevant when 
+naming the branches of your contributions (See section 3) so the Rucio maintainers 
```

Comment:
section 3?

Reply:
Section 3 of this page? Not sure how to better refer to it :-)